### DISCUSSION TITLE
Request for another pen thickness

### DISCUSSION THREAD

**Discussion by longtry** [xournalpp/xournalpp#4920] [Ideas]
On my device, I find the "thick" option too thin for my need, and the "extra thick" option too bulky. Could someone tell me their thickness in pixels? Anyway, I'd like to request a new option, or a plugin, of "quite thick" that falls between the 2 aforementioned.

If it's not too difficult, can I tinkle with Xournal++ files to get the desired pen size? I'm not a programmer, btw. If yes, please kindly guide me. Thank you.

**Comment by rolandlo** [comment]
The pen thicknesses are currently hardocoded in `src/core/control/ToolHandler.cpp`

https://github.com/xournalpp/xournalpp/blob/cf9477ed4da70227943660f24e24eda697ee1c8b/src/core/control/ToolHandler.cpp#L34-L42

You can compile your own version if you like changing these values. Otherwise you must wait that customizable thickness slider gets implemented, see PR  #3288 .

**Reply by longtry** [reply]
Thanks! I imagine that such a file could be opened and edited with Notepad, but can't be saved... What tools would I need to compile a .cpp file?

**Reply by longtry** [reply]
Is there any tentative schedule for custom thickness #3288?

**Comment by HRST221** [comment]
go to View>Toolbar>customize
From the dropbox drag Very Thick pen icon and place it on your toolbar

**Comment by x3oo** [comment]
i want to support this suggestion.

**Reply by searumblinguminari** [reply]
I too am looking forward to the day when I will be able to change the stroke thickness.

For now, I am adjusting the thickness by referring to the questions and answers in the link below.
https://github.com/xournalpp/xournalpp/discussions/5695#discussion-6693563

Thanks to everyone involved in development and all the users who are discussing it with me, I am able to use it comfortably. Thank you very much.
